src/zap/zapper.cpp

```diff
@@ -1515,6 +1519,12 @@ void Zapper::CompileAssembly(CORCOMPILE_NGEN_SIGNATURE * pNativeImageSig)
 
     // Close the file
     CloseHandle(hFile);
+
+    if (!WszMoveFileEx(strNativeImageTempPath.GetUnicode(), strNativeImagePath.GetUnicode(), MOVEFILE_REPLACE_EXISTING))
```

Comment:
I think it would be more logical to do this after the CloseHandle loop below.

Reply:
Done